### PR TITLE
Test Fixes: Miscellaneous fixes to help stablize tests

### LINKED ISSUE #230: Test Failure: Integration tests failing
This has been failing for the past few days for the same reason, starting with Commit e61a5f1...but that commit does not appear to be related to the failure. 

Travis log (while it lasts): https://travis-ci.org/logstash-plugins/logstash-input-beats/jobs/244957331

Copy of travis log:  https://gist.github.com/jakelandis/ed679c3d154fac7fd53feb15db3c94db

```
Finished in 10 minutes 35 seconds (files took 11.38 seconds to load)

17 examples, 2 failures, 1 pending
Failed examples:

rspec ./spec/integration/filebeat_spec.rb:271 # Filebeat TLS Server verification Client verification / Mutual validation CA root with multiples different CA, and unrelated CA appear first in the file successfully send the events

rspec ./spec/integration/filebeat_spec.rb:261 # Filebeat TLS Server verification Client verification / Mutual validation CA root intermediate create server and client certificate successfully send the events

Randomized with seed 47049
```

### PR DESCRIPTION
**Pull request by jakelandis**
* Bump file beat to filebeat-6.0.0-alpha2 for integration tests
* Error sooner if file beat fails to start
* Monkey patch fix for https://github.com/jordansissel/ruby-flores/issues/9 (certificate validation issue)
* Increase lifespan of certificates to 1 day to assist with troubleshooting (does not impact tests)

Fixes #230

Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

File: lib/tasks/test.rake [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
 VENDOR_PATH = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "vendor"))
 
+#TODO: Figure out better means to keep this version in sync
 if OS_PLATFORM == "linux"
-  FILEBEAT_URL = "https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-5.0.0-linux-x86_64.tar.gz"
+  FILEBEAT_URL = "https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-6.0.0-alpha2-linux-x86_64.tar.gz"
 elsif OS_PLATFORM == "darwin"
-  FILEBEAT_URL = "https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-5.0.0-darwin-x86_64.tar.gz"
+  FILEBEAT_URL = "https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-6.0.0-alpha2-darwin-x86_64.tar.gz"
 end
```
Comment (ph) [inline lib/tasks/test.rake]: There is a discussion to provide a link that describe the current artifact but its not done yet...

**Comment by ph** [comment]
LGTM

**Comment by jakelandis** [comment]
Merge still failed the test (despite passing here). I _think_ there is a let bleeding over to the wrong context when run in a specific order...but can not reproduce locally, even with same seed.  I added the let the context in question, at worst it is just duplicate code.

**Comment by jakelandis** [comment]
@ph - could i get another review ?

**Comment by ph** [comment]
> I believe that starting/stoping a TCP server to check for the open port, may prevent the org.logstash.beats.Server from properly starting. Anecdotal evidence shows that by removing the TCP server open port check allows for the Logstash Forwarder tests to succeed with a greater probablity.

I think this is a correct theory.

**Pull request closed without merging**